index.html

```diff
@@ -12,10 +12,71 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <header class="header"> <!-- contains logo and nav -->
+        <div class="container">   
+            <div class="header__inner">
+               <img src="img/karma-logo.svg" width="25" alt="karma-logo">
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+                <nav class="nav">
+                    <a  href="#">Meet Karma</a>
+                    <a  href="#">How it Works</a>
+                    <a  href="#">Store</a>
+                    <a  href="#">Blog</a>
+                    <a  href="#">Help</a>
+                    <a  href="#">Login</a>
+                </nav>
+            </div>
+        </div>
+    <!-- header starts here-->
+    </header>
+        <div class="hero">
+        <div class="container">
+            <h1>Introducing Karma</h1>
+            <h2>Bring WiFi with you, everywhere you go.</h2>
+            <button class="hero__button">Learn More</button>
+        </div>
+    </div>
+    <!-- Main starts here-->
+    <div class="main">
+        <div class="container">
+            <h1 class="main__title">Everyone needs a little Karma.</h1>
```

Comment:
This heading is slightly smaller than the hero heading, so it could be an h2

Reply:
Also, be careful - you added a very large top padding on all h1s in your CSS, so this heading is pushed down too low on the page.
How could you apply the top padding to the hero h1 only, and not this one?